github-pullrequest-plugin/src/test/resources/org/jenkinsci/plugins/github_integration/its/WorkflowITest/testContextStatuses.groovy

```diff
@@ -0,0 +1,31 @@
+package org.jenkinsci.plugins.github_integration.its.WorkflowITest
+
+
+try {
+    node('master') {
+        step([
+                $class       : 'GitHubPRStatusBuilder',
+                statusMessage: [
+                        content: "Run #${env.BUILD_NUMBER} started"
+                ]
+        ])
+        setGitHubPullRequestStatus state: 'PENDING', context: 'custom-context1', message: "Run #${env.BUILD_NUMBER} started"
```

Comment:
Minor... but if you happen to make another change perhaps one of these `setGitHubPullRequestStatus` calls should be made before the `node` block. This will help ensure that the method can be called outside of the node context.